lib/screens/v2/widget_instance/subway_status.ex

```diff
@@ -360,6 +360,9 @@ defmodule Screens.V2.WidgetInstance.SubwayStatus do
     Map.merge(%{route_pill: serialize_route_pill(route_id)}, serialize_alert(alert, route_id))
   end
 
+  @spec serialize_alert(Alert.t() | nil, String.t()) :: alert()
```

Comment:
We could use the type `Route.id()` here to be clear about the meaning (and same below).